docs/source/release-notes.md

```diff
@@ -4,6 +4,101 @@
 
 All published functionality in the Release Notes has been fully tested and verified with known limitations documented. To share feedback about this release, access our [NVIDIA Developer Forum](https://forums.developer.nvidia.com/).
 
+## TensorRT-LLM Release 1.1
+
+### Key Features and Enhancements
+
+- **Model Support**
+  - Add GPT-OSS model support
+  - Add Hunyuan-Dense model support
+  - Add Hunyuan-MoE model support
+  - Add Nemotron Nano VL V2 model support
+  - Add Seed-OSS model support
+- **Features**
+  - **KV Cache & Context:**
+    - **Connector API:** Introduced a new KV Cache Connector API for state transfer in disaggregated serving.
+    - **Reuse & Offloading:** Enabled KV cache reuse for MLA (Multi-Head Latent Attention) and added examples for host offloading.
+    - **Salting:** Implemented KV cache salting for secure cache reuse.
+  - **Speculative Decoding:**
+    - **Guided Decoding Integration:** Enabled guided decoding to work in conjunction with speculative decoding (including 2-model and draft model chunked prefill).
+    - **Eagle:** Added multi-layer Eagle support and optimizations.
+  - **Disaggregated Serving:**
+    - Added support for Guided Decoding in disaggregated mode.
+    - Optimized KV cache transfer for uneven pipeline parallelism.
+  - **Performance:**
+    - **DeepEP:** Optimized low-precision (FP4) combined kernels and all-to-all communication.
+    - **AutoTuner:** Refactored tuning config and generalized tactic selection for better kernel performance.
+    - **CuteDSL:** Integrated CuteDSL NVFP4 grouped GEMM for Blackwell.
+- **Benchmark**
+  - **New Benchmarks:**
+    - **Disaggregated Serving:** Added dedicated performance tests for disaggregated serving scenarios (`test_perf.py`).
+    - **Multimodal:** Enabled `benchmark_serving` support for multimodal models.
+    - **NIM:** Added specific performance test cases for NIM (NVIDIA Inference Microservices) integration.
+  - **Tooling Improvements:**
+    - **trtllm-bench:** Added support for sampler options, accurate device iteration timing, and improved data loading for benchmark datasets.
+    - **Metrics:** Enhanced reporting to include KV cache size metrics in benchmark results.
+    - **Scaffolding:** Added benchmark support for scaffolding examples.
+- **Documentation**
+  - **Deployment Guides:** Added comprehensive deployment guides for GPT-OSS, DeepSeek-R1, and VDR 1.0.
+  - **Feature Documentation:** Created new documentation for KV Cache Connector, LoRA feature usage, and AutoDeploy.
+  - **Tech Blogs:** Published blogs on "Combining Guided Decoding and Speculative Decoding" and "ADP Balance Strategy".
+  - **Hardware:** Updated documentation to reflect B300/GB300 support.
```

Comment:
Was this support added in this release? Was this just a docs update? Otherwise it might be confusing as this is listed under a documentation update.